website/frontend/src/components/pages/SendMessage/SendMessage.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useCallback, useEffect, useState, VFC } from 'react';
+import { useDispatch } from 'react-redux';
+import { useHistory, useParams } from 'react-router-dom';
+import { getTypeStructure, Metadata, parseHexTypes } from '@gear-js/api';
+import { MetaParam } from 'utils/meta-parser';
+import { Loader } from 'react-feather';
+import { RPC_METHODS } from 'consts';
+import ServerRPCRequestService, { RPCResponseError } from 'services/ServerRPCRequestService';
+import { GetMetaResponse } from 'api/responses';
+import { EventTypes } from 'types/events';
+import { AddAlert } from 'store/actions/actions';
+import { fileNameHandler } from 'helpers';
+import { MessageForm } from './children/MessageForm/MessageForm';
+import ArrowBack from 'assets/images/arrow_back.svg';
+import ProgramIllustration from 'assets/images/program_icon.svg';
+import './SendMessage.scss';
+
+type Params = { id: string };
+
+export const SendMessage: VFC = () => {
+  const dispatch = useDispatch();
+  const routeHistory = useHistory();
+  const routeParams = useParams<Params>();
+  const programId = routeParams.id;
+
+  const [meta, setMeta] = useState<Metadata | null>(null);
+  const [types, setTypes] = useState<MetaParam | null>(null);
+  const [ready, setReady] = useState(false);
+
+  const getMeta = useCallback(async () => {
+    const apiRequest = new ServerRPCRequestService();
+
+    return apiRequest.callRPC<GetMetaResponse>(RPC_METHODS.GET_METADATA, {
```

Comment:
It seems you've forgotten add `await` here
```suggestion
    return await apiRequest.callRPC<GetMetaResponse>(RPC_METHODS.GET_METADATA, {
```